src/main/resources/assets/creepy-creepers/lang/ta-in.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
file name must be underscores, not hyphens..

Reply:
#8